### DISCUSSION TITLE
germany_new bandplan

### DISCUSSION THREAD

**Discussion by betzburger** [AlexandreRouma/SDRPlusPlus#866] [General]
Good evening,

don't know if this is the right place. If not, Mr. Administrator, please move it to the right one.

I created a much more detailed bandplan of Germany and if you like, you can use it.

I tried to cover as much as possible from long wave up to 3 GHz without getting lost in too many details.
Sometimes a band is used from more than one service. In such a case I selected the most used services for this band. 
 
Have fun!

**Comment by AlexandreRouma** [comment]
there already are germany band plans so I recommend you open a pull request with the changes make to the existing ones.

**Comment by betzburger** [comment]
[germany_detail.json.zip](https://github.com/AlexandreRouma/SDRPlusPlus/files/11872761/germany_detail.json.zip)

**Reply by betzburger** [reply]
it is much more detailed and changes everything from existing ones.
